models/oneinch/arbitrum/oneinch_arbitrum_schema.yml

```diff
@@ -12,13 +12,13 @@ models:
         oneinch arbitrum calls & transfers
 
     columns:
-      - &blockchain
+      - &blockchain_not_null
```

Comment:
what is the intention for adding `_not_null` to the column name itself in the schema files?

typically, we see the columns in schema files match the models themselves, for documentation purposes. curious if you have a strong opinion on this approach?

Reply:
@jeff-dude `_not_null` columns contains "not null" test always. it's convenient in huge yml files where column can be null in one schema and can't be null in another.